src/main/java/com/thirdcc/webapp/service/impl/UserCCInfoServiceImpl.java

```diff
@@ -91,6 +99,83 @@ public UserCCInfoDTO save(UserCCInfoDTO userCCInfoDTO) {
             .map(this::clubFamilyDetails);
     }
 
+    @Override
+    @Transactional(readOnly = true)
+    public List<UserCCInfoDTO> getUserCCInfoByUserId(Long userId) {
+        log.debug("Request to get UserCCInfo of User : {}", userId);
+        List<UserCCInfoDTO> userCCInfoDTOList = userCCInfoRepository
+            .findAllByUserId(userId, Pageable.unpaged())
+            .map(userCCInfoMapper::toDto)
+            .getContent();
+        return getFullUserCCInfoList(userCCInfoDTOList, userId);
+    }
+
+    /**
+     * User CC Fish Level:
+     * First Year Session is "Junior Fish"
+     * Second Year Session is "Senior Fish"
+     * Later Year Session until Present is "Elder Fish"
+     *
+     * @param userCCInfoDTOList
+     * @param userId
+     * @return List of user CC Info Sorted by Year Session
+     */
+    private List<UserCCInfoDTO> getFullUserCCInfoList(List<UserCCInfoDTO> userCCInfoDTOList, Long userId) {
+        List<String> yearSessionWithCCFamilyRole = userCCInfoDTOList.stream()
```

Comment:
`yearSessionWithCCFamilyRole` is an inappropriate name. Since the list only contains yearsession string

Reply:
@kachun333 
this is the list of year session that user have CC Family Role
maybe I name it `yearSessionListHaveCCFamilyRole` ?

Reply:
resolved in d5253a35de87edf18588bc0c2bae3356b925f969